src/NMEA/ClimbInfo.hpp

```diff
@@ -45,6 +45,9 @@ struct OneClimbInfo
   /** Altitude gained while in the thermal [m]. May be negative. */
   double gain;
 
+  /** Altitude lost while in the thermal [m]. */
```

Comment:
I think this needs a better description; the semantics of this field should really be well-defined. Even from your code, I have difficulties deriving the meaning of this field (and I still don't know the practical use, and I believe it is a very fragile and inaccurate value, because it depends so much about the vario's update rate and the volatility of the TE altitude).

Reply:
Sure I will add some more words here.

Reply:
Did so